libbz2-rs-sys/src/compress.rs

```diff
@@ -51,13 +52,27 @@ impl<'a> LiveWriter<'a> {
         }
     }
 
-    fn write(&mut self, n: i32, v: u32) {
+    #[inline(always)]
+    fn flush_whole_bytes(&mut self) {
```

Comment:
Maybe don't outline this function instead? It has a single caller anyway.

Reply:
I like being able to give a name to the logic though

Reply:
In that case can you mark it as `#[inline]` instead?